package-linux_arm64.sh

```diff
@@ -53,13 +51,13 @@ $JAVA_HOME/bin/jpackage --input $BUILDDIR \
   --main-class $MAIN \
   --type app-image \
   --app-version $VERSION \
-  --java-options '--enable-preview'
- 
+  --java-options '--enable-preview -Dcom.sun.net.ssl.checkRevocation=false -Djava.security.revocation=false -Djava.security.egd=file:/dev/./urandom'
```

Comment:
I had to read up on the ssl checkRevocation flag and this turns the installer into an arbitrary code executing from network download back-door. I want the code to load faster too, but with the number of people using it, we need to be mindful of the security of the users too.